internal/testbed/linux-services/package-tests.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+# Copyright The OpenTelemetry Authors
+# Copyright Dynatrace LLC
+# SPDX-License-Identifier: Apache-2.0
+
+set -euov pipefail
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+REPO_DIR="$( cd "$SCRIPT_DIR/../../../../" && pwd )"
+export REPO_DIR
+PKG_PATH="${1:-}"
+DISTRO="dynatrace-otel-collector"
+
+SERVICE_NAME=$DISTRO
+PROCESS_NAME=$DISTRO
+
+# shellcheck source=scripts/package-tests/common.sh
+source "$SCRIPT_DIR"/common.sh
+
+if [[ -z "$PKG_PATH" ]]; then
+    echo "usage: ${BASH_SOURCE[0]} DEB_OR_RPM_PATH" >&2
+    exit 1
+fi
+
+if [[ ! -f "$PKG_PATH" ]]; then
+    echo "$PKG_PATH not found!" >&2
+    exit 1
+fi
+
+
+pkg_base="$( basename "$PKG_PATH" )"
+pkg_type="${pkg_base##*.}"
+if [[ ! "$pkg_type" =~ ^(deb|rpm)$ ]]; then
+    echo "$PKG_PATH not supported!" >&2
+    exit 1
+fi
+image_name="otelcontribcol-$pkg_type-test"
+container_name="$image_name"
+container_exec="podman exec $container_name"
+
+trap 'podman rm -fv $container_name >/dev/null 2>&1 || true' EXIT
+
+podman build -t "$image_name" -f "$SCRIPT_DIR/Dockerfile.test.$pkg_type" "$SCRIPT_DIR"
+podman rm -fv "$container_name" >/dev/null 2>&1 || true
+
+podman run --name "$container_name" -d "$image_name"
+podman_cp "$container_name" internal/testbed/linux-services/config.test.yaml /etc/dynatrace-otel-collector/config.yaml
+install_pkg "$container_name" "$PKG_PATH"
+
+# ensure service has started and still running after 5 seconds
+sleep 5
+echo "Checking $SERVICE_NAME service status ..."
+$container_exec systemctl --no-pager status "$SERVICE_NAME"
+
+echo "Checking $PROCESS_NAME process ..."
+$container_exec pgrep -f -a -u otel "$PROCESS_NAME"
+
+# test uninstall
+echo
```

Comment:
I guess something should be printed out here, right?

Reply:
this came from upstream, i think we should just remove it

Reply:
```suggestion
```